src/main/java/org/folio/innreach/domain/service/impl/InnReachTransactionActionServiceImpl.java

```diff
@@ -552,4 +571,8 @@ private InventoryItemDTO fetchItemByBarcode(String itemBarcode) {
     return itemService.findItemByBarcode(itemBarcode)
       .orElseThrow(() -> new IllegalArgumentException("Item is not found by barcode: " + itemBarcode));
   }
+
+  private boolean checkLoanActionAndStatus(String loanAction, String loanStatus) {
```

Comment:
the name of the method should explain what condition is being checked. This one doesn't give any hint what the action or status is expected

Reply:
renamed